src/proto/ctl/storage_nvme.proto

```diff
@@ -34,25 +34,27 @@ message NvmeController {
 	// Namespace represents a namespace created on an NvmeController.
 	message Namespace {
 		int32 id = 1;		// namespace id
-		int32 capacity = 2;	// device capacity in GBytes
+		int32 size = 2;	// device capacity in GBytes
+		string ctrlrpciaddr = 3;
 	}
 
 	message Health {
 		uint32 temp = 1;
-		uint32 tempwarn = 2; // temperature warning duration
-		uint32 tempcrit = 3; // temperature critical duration
-		uint64 ctrlbusy = 4; // controller busy duration
+		uint32 tempwarntime = 2; // temperature warning duration
+		uint32 tempcrittime = 3; // temperature critical duration
+		uint64 ctrlbusytime = 4; // controller busy duration
 		uint64 powercycles = 5;
 		uint64 poweronhours = 6;
 		uint64 unsafeshutdowns = 7;
 		uint64 mediaerrors = 8;
-		uint64 errorlogs = 9; // error log entries
+		uint64 errorlogentries = 9; // error log entries
 		// critical warnings
-		bool tempwarning = 10;
-		bool availspare = 11;
-		bool reliability = 12;
-		bool readonly = 13;
-		bool volatilemem = 14;
+		bool tempwarn = 10;
+		bool availsparewarn = 11;
+		bool reliabilitywarn = 12;
+		bool readonlywarn = 13;
+		bool volatilewarn = 14;
+		string ctrlrpciaddr = 15;
```

Comment:
Any reason the names are all lowercase instead of camelcase?

Reply:
I was just working with what was already there. Didn't really think about it, tbh.